tests/integration/Context/BootlegServer.php

```diff
@@ -107,6 +107,9 @@ private function runHttpServer(string $address, string $requestFile, string $res
             throw new \RuntimeException("Port probably taken");
         }
 
+        // wait for a moment, so that the server has a chance to start
+        sleep(1);
```

Comment:
It was being done only once, so it's not a big delay, but I agree that it's not a great implementation, so I've replaced the sleep call with some logic to wait for the server to start.

Reply:
That's actually not true. It is called on each construct of the context class which is initialized between each feature/scenario. I can't find the actual documentation for it but please note that [feature hooks are registered as static methods](https://docs.behat.org/en/v2.5/guides/3.hooks.html#feature-hooks) because an instance don't exists whereas [scenario hooks](https://docs.behat.org/en/v2.5/guides/3.hooks.html#scenario-hooks) are registered as regular methods (Aka we have an instance)

It's not that important - I will approve it with or without this 👍